src/CFixer/main.cpp

```diff
@@ -26,5 +28,10 @@ int main(int argc, char *argv[]) {
         return 0;
     }
 
+    bool runProc = worker.run();
+    if (!runProc) {
+        return 0;
```

Comment:
Why you return non-rrror code if  the run method is failed?
```suggestion
        return 1;
```